### PR TITLE
[DJFS-27] Develop an E-commerce Store Database

### PR DESCRIPTION
**Pull request by opeyemipeter1759**
## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

**Comment by opeyemipeter1759** [comment]
@adaugochi Kindly review my PR thanks

File: submissions/opeyemiPeter/database/init.sql [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+select * from Orders;
+
+-- 7. Order Item
+-- creation of the Table
+create table orderItem(
+ orderItemId int not null auto_increment,
+ OrderId int not null,
+ productId int not null,
+ quantity int not null,
+ unitPrice int not null,
```
Comment (adaugochi) [inline submissions/opeyemiPeter/database/init.sql:160]: kindly use `decimal` for anything relating to amount

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/opeyemiPeter/database/init.sql (hunk 1 above)
Comment (opeyemipeter1759) [reply submissions/opeyemiPeter/database/init.sql]: @ i have changed it.

Comment (opeyemipeter1759) [reply submissions/opeyemiPeter/database/init.sql]: @adaugochi

Comment (opeyemipeter1759) [reply submissions/opeyemiPeter/database/init.sql]: @adaugochi I have used decimal as directed. thanks

**Review by adaugochi — APPROVED** [review]

**Pull request merged**